src/modules/navigator/takeoff.cpp

```diff
@@ -72,17 +72,11 @@ Takeoff::on_active()
 
 		position_setpoint_triplet_s *pos_sp_triplet = _navigator->get_position_setpoint_triplet();
 
-		// set loiter item so position controllers stop doing takeoff logic
```

Comment:
Can a takeoff item even be reached if landed? This may need some hardening again.